src/main/java/racingcar/validate/Validation.java

```diff
@@ -0,0 +1,51 @@
+package racingcar.validate;
+
+import racingcar.constants.ErrorType;
+import racingcar.constants.GameConstants;
+
+import java.util.Set;
+
+public class Validation {
+
+    public static String validateCreateCar(String input, Set<String> distinctCarsName) {
+        String carName = input.trim();
+        validateInput(carName);
+        validateNameLength(carName);
+        validateDuplicateCarName(carName, distinctCarsName);
+        return carName;
+    }
+
+    public static int validateTryGame(String count) {
+        try {
+            validateGameTryCount(Integer.parseInt(count));
+            return Integer.parseInt(count);
```

Comment:
```suggestion
            int count = Integer.parseInt(input);
            validateGameTryCount(count);
            return count;
```
중복된 처리를 다음과 같이 변경해볼 수 있을 것 같아요~